org.eclipse.scanning.device.ui/src/org/eclipse/scanning/device/ui/util/ScanRegions.java

```diff
@@ -80,6 +81,7 @@ public static final BoundingBox createBoxFromPlot(Object model) throws Generator
 
 	private static final BoundingBox bounds(List<IROI> rois) {
 		
+		rois = Optional.of(rois).orElse(Collections.emptyList());

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "rois". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1226)